sdk/src/main/java/com/kin/ecosystem/history/view/OrderHistoryFragment.java

```diff
@@ -37,9 +37,13 @@ public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container,
 	}
 
 	@Override
-	public void onDestroyView() {
-		super.onDestroyView();
-		orderHistoryPresenter.onDetach();
+	public void onDestroy() {
+		if (orderHistoryPresenter != null) {
```

Comment:
don't see a particular reason to null out presenter, the leak direction is the opposite when presenter holds the activity, isn't it?

Reply:
first of all release memory it's a good approach :), but you are right the leak direction is the opposite.